db/migrate/20170919084512_new_volunteer_state_fields.rb

```diff
@@ -0,0 +1,12 @@
+class NewVolunteerStateFields < ActiveRecord::Migration[5.1]
+  def change
+    change_table :volunteers do |t|
+      t.integer :acceptance, default: 0
+      t.datetime :accepted_at
+      t.datetime :rejected_at
+      t.datetime :resigned_at
+      t.datetime :undecided_at
+      t.boolean :take_more_assignments, default: false
+    end
```

Comment:
1. Do we still need the old `state` attribute?
2. Do we need to convert the old `state` into the new enum?

Reply:
No we don't, But its WIP